libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -37,6 +37,9 @@ export class ButtonComponent {
     @Input()
     type: ButtonType;
 
+    @Input()
+    fdType: ButtonType;
```

Comment:
dont think this should be included in this pr

Reply:
it should be there